Add next steps for Crescend get-started (#30)
MicrosoftDocs/PowerShell-Docs-Modules: reference/docs-conceptual/Crescendo/overview.md

---
description: Overview of the Crescendo module
ms.date: 03/07/2022
title: Crescendo overview
---
# Microsoft.PowerShell.Crescendo overview

> PowerShell is capable of invoking command-line tools like any shell. However, it would improve the

Reviewer: ```suggestion
PowerShell is capable of invoking command-line tools like any shell. However, the experience can be
```

> PowerShell, like other shells, is capable of invoking command-line tools like any shell. However, it

Reviewer 2: ```suggestion
PowerShell, like other shells, is capable of invoking command-line tools. However, it
```

> experience if the command-line tool could participate in the PowerShell pipeline and take advantage

Reviewer: ```suggestion
improved if the command-line tool participates in the PowerShell pipeline and takes advantage
```
